elasticdl/python/master/pod_manager.py

```diff
@@ -352,16 +352,20 @@ def _process_worker(self):
 
     def _start_worker(self, worker_id):
         logger.info("Starting worker: %d" % worker_id)
-        bash_command = self._worker_args[1]
+        # self._worker_args has 2 strings. The first string is "-c" and
+        # the second string is the shell command to run, like
+        # ["-c", "python -m elasticdl.python.worker.main --minibatch_size 64"]
+        job_command = self._worker_args[1]
         if self._ps_addrs:
```

Comment:
We can extract L358 - L367 into a separate method: _complement_job_command

Reply:
Good suggestion.